include/envoy/http/codec.h

```diff
@@ -260,6 +284,16 @@ class ClientConnection : public virtual Connection {
    * @return StreamEncoder& supplies the encoder to write the request into.
    */
   virtual StreamEncoder& newStream(StreamDecoder& response_decoder) PURE;
+
+  /**
+   * Called when the connection goes over its high watermark.
+   */
+  virtual void onBelowWriteBufferLowWatermark() PURE;
+
+  /**
+   * Called when the connection goes from over its high watermark to under its low watermark.
```

Comment:
Are these comments the opposite way around (or maybe I'm standing on my head today ;) )?